moVirt/src/main/java/org/ovirt/mobile/movirt/ui/vms/VmDetailGeneralFragment.java

```diff
@@ -225,7 +225,6 @@ public void onLoaderReset(Loader<Cursor> loader) {
         // do nothing
     }
 
-    @UiThread
     public void renderVm(Vm vm) {
         if (!vm.isSnapshotEmbedded()) {
             getActivity().setTitle(String.format(VM_DETAILS, vm.getName()));
```

Comment:
This is safe if we call these functions from onLoadFinished which was [initialized](http://stackoverflow.com/questions/30825534/which-threads-call-oncreateloader-and-onloadfinished) in UiThread. This is exactly what we do (We call init from @AfterViews), which is called from UiThread.